packages/button/src/vaadin-button.d.ts

```diff
@@ -4,10 +4,7 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
-import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
-import { ActiveMixin } from '@vaadin/component-base/src/active-mixin.js';
-import { FocusMixin } from '@vaadin/component-base/src/focus-mixin.js';
-import { TabindexMixin } from '@vaadin/component-base/src/tabindex-mixin.js';
+import { ActiveMixin, ElementMixin, FocusMixin, TabindexMixin } from '@vaadin/component-base';
```

Comment:
Let's keep using individual imports as we do elsewhere.

```suggestion
import { ActiveMixin } from '@vaadin/component-base/src/active-mixin.js';
import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
import { FocusMixin } from '@vaadin/component-base/src/focus-mixin.js';
import { TabindexMixin } from '@vaadin/component-base/src/tabindex-mixin.js';
```

Please update `.d.ts` file accordingly.